### DISCUSSION TITLE
How do we handle runtime dependencies while installing pax files? More details below

### DISCUSSION THREAD

**Discussion by HarithaIBM** [zopencommunity/meta#576] [Ideas]
Say, i have gpg tool that has runtime dependency on pinentry. Now if use `zopen install gpg` it is taken care of. Whereas, running `.env` of the pax file doesnot. So, do we handle this in the setup script (as discussed with @IgorTodorovskiIBM  yesterday :) ) ? Any  thoughts/ideas here please - @MikeFultonDev @IgorTodorovskiIBM @DevonianTeuchter ?

**Comment by MikeFultonDev** [comment]
Is this runtime dependency different than git having a runtime dependency on bash?

**Reply by HarithaIBM** [reply]
Similar.

**Comment by v1gnesh** [comment]
Perhaps by pulling the dependent pax files for download alongside the main one.
Which means a download page may be needed, so we can guarantee deps are also downloaded.
Or, pack gpg with pinentry & other deps as a collection pax.

**Comment by DevonianTeuchter** [comment]
Since we know the dependencies a package needs when building the release, what if release notes (or README.md) for that package were autogenerated with a list to the latest release of any packages that were depended on? It would still be a manual download process but everything would be right there for a user to click on...

**Reply by IgorTodorovskiIBM** [reply]
Runtime dependencies are listed here currently: https://github.com/ZOSOpenTools/gitport/releases
One simple solution is to make them clickable.

**Reply by DevonianTeuchter** [reply]
> Runtime dependencies are listed here currently: https://github.com/ZOSOpenTools/gitport/releases
> One simple solution is to make them clickable.

Yep, that's what I meant - rather than be the current flat list, have them clickable
